packages/web-components/src/tablist/tablist.styles.ts

```diff
@@ -0,0 +1,216 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '../utils/index.js';
+import {
+  borderRadiusCircular,
+  colorCompoundBrandForeground1Hover,
+  colorNeutralForeground1,
+  colorNeutralForeground1Hover,
+  colorNeutralForeground2,
+  colorNeutralForegroundDisabled,
+  colorSubtleBackgroundHover,
+  colorSubtleBackgroundPressed,
+  curveDecelerateMax,
+  durationSlow,
+  fontSizeBase300,
+  fontSizeBase400,
+  lineHeightBase300,
+  lineHeightBase400,
+  spacingHorizontalMNudge,
+  spacingHorizontalSNudge,
+  spacingVerticalL,
+  spacingVerticalMNudge,
+  spacingVerticalS,
+  spacingVerticalSNudge,
+  spacingVerticalXXS,
+  strokeWidthThicker,
+} from '../theme/design-tokens.js';
+
+/**
+ * @public
+ */
+export const styles = css`
+  ${display('flex')}
+
+  :host {
+    --tabPaddingInline: inherit;
+    --tabPaddingBlock: inherit;
+    --tabIndicatorInsetInline: 0;
+    --tabIndicatorInsetBlock: 0;
+    box-sizing: border-box;
+    color: ${colorNeutralForeground2};
+    flex-direction: row;
+  }
+
+  :host([orientation='vertical']) {
+    flex-direction: column;
+  }
+
+  :host ::slotted([role='tab']) {
+    align-items: flex-start;
+  }
+
+  /* indicator animation  */
+  :host ::slotted([role='tab'][data-animate='true'])::after {
```

Comment:
I wonder if we should be specific about the tab element type, i.e. instead of using `[role='tab']`, use `fluent-tab`, because if an author decides to use a different element, this may break their element's styles.

Reply:
In the `fluent-tabs` it is `fluent-tab` but my reasoning for something more generic was that consumers might recompose `fluent-tab` with a different prefix (I saw `fluent-beta-` the other day). So using `role` (hopefully) makes it a bit more agnostic to the component registration. Open to ideas here.

Reply:
I think the only concern I have here is that when this goes to element internals the role will not reflect. Agree on the flexibility here, I wonder...does it work to reference `FluentDesignSystem.prefix`? I don't think we've gotten that addressed to allow teams to change the prefix properly yet, but that _could_ in theory work?

Reply:
Ah I didn't know there could be `fluent-beta-` elements. In that case, maybe `::slotted([slot="tab"])` can solve Chris' concern over `role` not being exposed due to element internals.

Reply:
I think the tension here is that we would need to then have these _not_ be the default slot, which would be weird ergonomically as the tabs are the most intuitive and obvious content to be expected as the default slots.

Perhaps, given that the outliers would be `start` and `end` slots, we look at `*:not()`? Not ideal, but scoped within the shadow root and not _every_ element?

Reply:
Some good news maybe, I removed the `start/end` slots on tablist, so we could maybe just do `::slotted(*)`. Unless we need `start/end`.. then it's bad news.

Reply:
I was trying to understand this earlier but haven't figured it out: if the tab elements are assigned to the default slot, how does this work? https://github.com/microsoft/fluentui/blob/1931b54c1517fae68a004022fad736c1c06168de/packages/web-components/src/tablist/tablist.ts#L126-L142

Reply:
So `slot="tab"` and `role="tab"` get set by the `Tab` component and `Tablist` does not have a default slot, only `<slot name="tab">`.

https://github.com/microsoft/fluentui/blob/1931b54c1517fae68a004022fad736c1c06168de/packages/web-components/src/tab/tab.template.ts#L7

Reply:
Ah I see. I was confused why there was no `slot="tab"` on the `<fluent-tab>` elements in the stories 😄

Reply:
Ah, so `::slotted([slot="tab"]) {}`